src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -431,3 +432,92 @@ const struct pcm_func_map pcm_func_map[] = {
 const size_t pcm_func_count = ARRAY_SIZE(pcm_func_map);
 
 #endif
+static int pcm_convert_s16_c32_to_s32_c32(const struct audio_stream *source,
+					  uint32_t ioffset, struct audio_stream *sink,
+					  uint32_t ooffset, uint32_t samples)
+{
+	uint32_t buff_frag = 0;
+	int32_t *src;
+	int32_t *dst;
```

Comment:
Is there a reason why these can not be in the same line?

Reply:
sure, thanks. I will update it.

Reply:
@lyakh, for me they are:
```
int32_t *src;
int32_t *dst;
```